ldk-server/src/main.rs

```diff
@@ -103,6 +105,15 @@ fn main() {
 
 	let event_publisher: Arc<dyn EventPublisher> = Arc::new(NoopEventPublisher);
 
+	#[cfg(feature = "events-rabbitmq")]
+	let event_publisher: Arc<dyn EventPublisher> = {
+		let rabbitmq_config = RabbitMqConfig {
+			connection_string: config_file.rabbitmq_connection_string,
+			exchange_name: config_file.rabbitmq_exchange_name,
+		};
+		Arc::new(RabbitMqEventPublisher::new(rabbitmq_config))
+	};
```

Comment:
Should we open a connection to ensure the config parameters are valid?

Reply:
Makes sense, though how would an operator be aware of the rabbitmq unavailability? IIUC, unavailability would cause the event queue to be backed up. The underlying node would still progress, but from the user's perspective the payment store would be stale since event publishing failure would prevent persisting the payment.

This doesn't need to hold up the PR, but just wondering how this would be monitored.